app/shared/text-editor/styles.css

```diff
@@ -0,0 +1,120 @@
+body {
```

Comment:
Any reason we can't use design-system instead?

Reply:
these are demo styles, but let me try and figure out how we can have the entire text editor get the same styles we will have on canvas

Reply:
we neeed to setup global styles for all stories, but otherwise this file is gone